Example/API.swift

```diff
@@ -12,43 +12,27 @@ import Fetch
 
 public struct API {
     
-    public struct BlogPosts {
-        
-        public static func list() -> Resource<[BlogPost]> {
-            return Resource(
-                method: .get,
-                path: "/posts")
-        }
-        
-        public static func detail(id: Int) -> Resource<BlogPost> {
-            return Resource(
-                method: .get,
-                path: "/posts/\(id)")
-        }
-        
-        public static func create(_ post: BlogPost) -> Resource<BlogPost> {
+    static func setup() {
+        let config = Fetch.Config(baseURL: URL(string: "https://yoururlhere.com")!)
+        APIClient.shared.setup(with: config)
+    }
+    
+    struct Basics {
+        static func organization(with username: String) -> Resource<Organization> {
             return Resource(
-                method: .post,
-                path: "/posts",
-                body: [
-                    "title": post.title,
-                    "author": post.author
-                ])
+                baseURL: URL(string: "https://api.github.com")!,
+                path: "orgs/\(username)")
         }
         
-        public static func nestedTest() -> Resource<BlogPost> {
+    }
+    
+    struct Stubbing {
```

Comment:
having a separate "Stubbing" resources makes it seem like that stubbed resources have to be handled differently than normal resources.